hathor/transaction/base_transaction.py

```diff
@@ -1035,6 +1035,7 @@ def to_json(self, decode_script: bool = False, include_metadata: bool = False) -
         data['timestamp'] = self.timestamp
         data['version'] = int(self.version)
         data['weight'] = self.weight
+        data['signal_bits'] = self.signal_bits
```

Comment:
Should we serialize it as a string of zeros and ones?

Reply:
I don't see a reason why, what are your thoughts?

Reply:
I feel it's weird to get an integer number. But it's ok too.